app/views/staff/proposals/_form.html.haml

```diff
@@ -12,18 +12,10 @@
         = f.text_field :slides_url, class: "form-control"
   .row
```

Comment:
looks like this entire row is only for tags so you can move the tag check to here